### ISSUE TITLE
respect Cache Directory Tagging Specification

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#3659]
Imho it would make sense if we respect https://bford.info/cachedir/

It basically proposes that our cache dir should have a file CACHEDIR.TAG with a content like:

```
Signature: 8a477f597d28d172789f06886806bc55
# This file is a cache directory tag created by Elektra, see
# https://www.libelektra.org
```

**Comment by mpranj** [comment]
Mozilla interestingly decided against implementing [this](https://bugzilla.mozilla.org/show_bug.cgi?id=252179).

I suppose we can add support though, since there should be no implementation hurdles.

**Comment by markus2330** [comment]
The decision of Mozilla looks political to me, technically their argumentation is wrong. XDG_CACHE_HOME actually introduces the problem, as it gives you no guarantee that `~/.cache` is used or any other directory. This is also explained in https://bford.info/cachedir/

Anyway, it is low priority but it should be easy to implement. We can also offer this issue to be implemented for the CM course? Can you point out where this would be best implemented?

**Comment by markus2330** [comment]
@mpranj I think we should clean up a bit and not take too many issues from release to release. Can you somehow realistically determine which of your issues will actually be done until the next release (Juli), for the others either remove the milestone or add `0.9.*` (if it must be before 1.0). If you need help in fixing some of the issues, please ask. I think we are already in a state where we actually should fix one issue after the other.

**Comment by mpranj** [comment]
> I think we should clean up a bit and not take too many issues from release to release. 

Sounds good. I also don't think I will fix this issue here for 0.9.7.

----

To be frank, some issues were simply not as important _to me_ as getting the GSettings update out. The GSettings fix will land in 0.9.7, still experimental as before, but it will work. (I use it on my main machine .. )

**Comment by markus2330** [comment]
>  Sounds good. I also don't think I will fix this issue here for 0.9.7.

And maybe not even 1.0, that is why I did not tag it for 0.9.*.

> To be frank, some issues were simply not as important to me as getting the GSettings update out.

I fully agree, I don't think we need to queue them up for the next release. Imho the milestones should help us to not forget important issues.

> The GSettings fix will land in 0.9.7, still experimental as before, but it will work. (I use it on my main machine .. )

I am looking forward! :rocket:
